alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -879,24 +879,27 @@ impl Canvas {
     /// `self.width` and an attached rectangle to form a "╭" using a given `stroke_size` in the
     /// bottom-right quadrant of the `Canvas` coordinate system.
     fn draw_rounded_corner(&mut self, stroke_size: usize) {
-        let r = 0.5 * (self.width.min(self.height) + stroke_size) as f32;
+        let r = (self.width.min(self.height) + stroke_size) as f32 / 2.;
         let w = stroke_size as f32;
 
-        let mut x_offset = 0.0;
-        let mut y_offset = 0.0;
-        let d_bias: f32;
-        if self.height > self.width {
-            y_offset = self.height as f32 * 0.5 - r + w * 0.5;
-            if (self.width % 2 != self.height % 2) && (self.height % 2 == stroke_size % 2) {
-                y_offset += 1.0;
+        let bigger = self.width.max(self.height);
+        let smaller = self.width.min(self.height);
+
+        let d_bias = if smaller % 2 == stroke_size % 2 { 0. } else { 0.5 };
+
+        let mut x_offset = 0.;
+        let mut y_offset = 0.;
```

Comment:
Generally, declaration should be closer to a place where something meaningful is assigned.